src/app/main/component/shared/components/warning-pop-up/warning-pop-up.component.scss

```diff
@@ -26,40 +27,67 @@
   font-weight: 500;
   font-size: 21px;
   line-height: 28px;
+  padding-bottom: 1em;
   color: var(--secondary-dark-grey);
 }
 
 .buttons-container {
   display: flex;
+  flex-flow: row;
+  max-width: fit-content;
+  align-items: center;
 }
 
+.ubs-primary-global-button,
 .primary-global-button,
-.secondary-global-button {
+.secondary-global-button,
+.ubs-secondary-global-button,
+.red {
   font-family: var(--primary-font);
   font-size: 16px;
   font-weight: 600;
   line-height: 16px;
-  padding: 16px 40px;
+  padding: 14px 36px;
 }
 
-.secondary-global-button {
+.red {
+  background: var(--ubs-error-ligh-red);
+  margin-right: 0;
+}
+
+.red:hover {
+  background: var(--secondary-dark-grey);
+}
```

Comment:
You can use scss nesting like `&:hover { ... }` and just move it to the block above

Reply:
ok. Thanks I'll fix it local)

Reply:
Done